src/ConfigCatClient.php

```diff
@@ -541,7 +541,7 @@ private function getSettingsResult(): SettingsResult
     private function getRemoteSettingsResult(): SettingsResult
     {
         $cacheEntry = $this->cache->load($this->cacheKey);
-        if (!$this->offline && $cacheEntry->getFetchTime() + $this->cacheRefreshInterval < time()) {
+        if (!$this->offline && $cacheEntry->getFetchTime() + $this->cacheRefreshInterval < Utils::getUnixMilliseconds()) {
```

Comment:
While `getFetchTime` returns msec now, `cacheRefreshInterval` is a second-based value, isn't it?

Reply:
👍